.github/workflows/ci.yml

```diff
@@ -75,6 +75,8 @@ jobs:
           SLACK_SIGNING_SECRET: ${{ secrets.SLACK_SIGNING_SECRET }}
           SLACK_BOT_DATA_PATH: ${{ secrets.SLACK_BOT_DATA_PATH }}
           OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
+          CLIENT_ID: ${{ secrets.CLIENT_ID }}
+          CLIENT_SECRET: ${{ secrets.CLIENT_SECRET }}
```

Comment:
Name need to specify to which service these secrets are related.

Reply:
I'm not sure if you figured it out already. You will also need to add these secrets in the repo. But you probably don't have access so @ihorsikora or I can do it for you.

Reply:
Both client_id and client_secret were added for token rotation, but then I figured out that this rotation method is related to the bot access token, not the app config one, so I will remove them from workflow in the next commit

Reply:
@AlexEmets please remove this if not needed and we can merge this PR